libsql/internal/http/hranaV2/hranaV2.go

```diff
@@ -286,7 +287,119 @@ func sendPipelineRequest(ctx context.Context, msg *hrana.PipelineRequest, url st
 	return result, false, nil
 }
 
+func (h *hranaV2Conn) executeMsg(ctx context.Context, query string, msg *hrana.PipelineRequest) (*hrana.PipelineResponse, error) {
+	result, err := h.sendPipelineRequest(ctx, msg, false)
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute SQL: %s\n%w", query, err)
+	}
+
+	for _, r := range result.Results {
+		if r.Error != nil {
+			return nil, fmt.Errorf("failed to execute SQL: %s\n%s", query, r.Error.Message)
+		}
+		if r.Response == nil {
+			return nil, fmt.Errorf("failed to execute SQL: %s\n%s", query, "no response received")
+		}
+	}
+	return result, nil
+}
+
+func (h *hranaV2Conn) executeInChunks(ctx context.Context, query string, wantRows bool) (*hrana.PipelineResponse, error) {
+	iterator := sqliteparserutils.CreateStatementIterator(query)
+
+	stmt, _, isEOF := iterator.Next()
+	// Skip all empty statements
+	for {
+		if stmt != "" {
+			break
+		}
+		stmt, _, isEOF = iterator.Next()
+		if isEOF {
+			break
+		}
+	}
+	if isEOF {
+		msg := &hrana.PipelineRequest{}
+		executeStream, err := hrana.ExecuteStream(stmt, nil, wantRows)
+		if err != nil {
+			return nil, fmt.Errorf("failed to execute SQL: %s\n%w", query, err)
```

Comment:
I think we should not log `query` as it can be veeeery huge (over 20MB already)

Reply:
Fair point. Will remove.

Reply:
Actually, on a second thought, I think it's better to keep logging the query. It's big - yes - but it's still better to see a big query and be able to say what's wrong with it instead of having nothing. Especially on production servers that is super valuable to see the failing query.

Reply:
Hm...that's true - but users can always add this logging on their side, no?
And in some cases you don't want to see this logs, right? For example, in turso-cli you don't want to see whole dump in the console again - you know that you are loading it. 

Plus, huge logs still not very nice thing to have - they can make logs not so usable (especially if you don't have new lines in the file, because almost all viewers/editors struggle with such payloads) and also can create problems like logs flooding (which can lead to disk overflow or truncation of really helpful logs).

Reply:
It is true that a user can add their own logging. On the other hand, many users come and complain about the error and if there's enough info in the error for us to solve the problem then it means orders of magnitude less support work as opposed to explaining to user where they have to add logging, get the issue again and report the logs back to us. I need to think a bit more about what's worth more.

Reply:
Ok, maybe then we can add current chunk in the logs + more information about progress (chunk index, time elapsed, etc)? This will reduce log size and also should increase detalization of logs...

Reply:
I think I will just remove it for now. We might want to think about some sort of verbose mode that's easy to turn on for users later.